### DISCUSSION TITLE
Mapping a field with both attribute and value

### DISCUSSION THREAD

**Discussion by karthikjeyaram** [atlasmap/atlasmap#4014] [Q&A] [answered]
Dear Community, Considering the below example, if we load the below xml in the mapper, where we have both attribute (id) and value (Orange) for itemId tag. We are able to map only the attribute (id) here, but not able to map the value (i.e: Orange, Apple). Is there a way to map both value and attribute of an element? Thank you.

`  <items>
    <item>
      <itemId id="1">Orange</itemId>
    </item>
    <item>
      <itemId id="2">Apple</itemId>
    </item>
  </items>`

**Comment by igarashitm** [comment, marked as answer]
You'd need to wait for https://github.com/atlasmap/atlasmap/issues/3785 to be implemented. Contributions are always welcome!
